vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1564,3 +1565,70 @@ def fused_moe(
                          a1_scale=a1_scale,
                          a2_scale=a2_scale,
                          block_shape=block_shape)
+
+
+def cutlass_moe(
+    a_q: torch.Tensor,
+    a_scale: torch.Tensor,
+    w1_q: torch.Tensor,
+    w2_q: torch.Tensor,
+    w1_scale: torch.Tensor,
+    w2_scale: torch.Tensor,
+    topk_weights: torch.Tensor,
+    topk_ids: torch.Tensor,
+    m: int,
+    n: int,
+    k: int,
+    num_groups: int,
+    ab_strides1: torch.Tensor,
+    c_strides1: torch.Tensor,
+    ab_strides2: torch.Tensor,
+    c_strides2: torch.Tensor,
+    intermediate_scale: Optional[torch.Tensor] = None,
+):
+    topk = topk_ids.shape[1]
+    per_act_token = a_scale.numel() != 1
+    device = a_q.device
+
+    expert_offsets = torch.empty((num_groups + 1),
+                                 dtype=torch.int32,
+                                 device=device)
+    problem_sizes1 = torch.empty((num_groups, 3),
+                                 dtype=torch.int32,
+                                 device=device)
+    problem_sizes2 = torch.empty((num_groups, 3),
+                                 dtype=torch.int32,
+                                 device=device)
+
+    a_map = torch.empty((topk_ids.numel()), dtype=torch.int32, device=device)
+    c_map = torch.empty((topk_ids.numel()), dtype=torch.int32, device=device)
+
+    torch.ops._C.get_grouped_mm_data(topk_ids, expert_offsets, problem_sizes1,
+                                     problem_sizes2, a_map, c_map, num_groups,
+                                     n, k)
```

Comment:
Add wrappers for `get_grouped_mm_data` and `cutlass_grouped_mm` in `vllm/_custom_ops.py`?